libs/resea/malloc.c

```diff
@@ -3,9 +3,11 @@
 #include <resea/printf.h>
 #include <string.h>
 
+#define NUM_CHUNKS 16
 extern char __heap[];
 extern char __heap_end[];
-static struct malloc_chunk *chunks = NULL;
+
+static struct malloc_chunk *chunks[NUM_CHUNKS];
```

Comment:
How about renaming `chunks` and `NUM_CHUNKS` to `bins` and `NUM_BINS` respectively? I think *bin* is more appropriate name for them.

Reply:
I agree. I have made the change.